packages/driver/src/cy/commands/index.ts

```diff
@@ -4,8 +4,6 @@ import * as Agents from './agents'
 
 import * as Aliasing from './aliasing'
 
-import * as Angular from './angular'
```

Comment:
`cy.ng()` is undocumented and old, and consulting with Chris and Brian, we've decided to remove it. This is technically a breaking change, but I haven't been able to find anyone in the wild using it.

I'm sure  there's someone, but... :woman_shrugging:

Reply:
Should we keep a stubbed version around that throws an error explaining it's been removed and they should use component testing?

Reply:
Personally I don't think so. `.ng()` doesn't show up in our docs anywhere, not even a passing mention in the changelog. I don't think it's worth keeping any form of backwards compatibility (even an error message) beyond mentioning this change as breaking in Cy 11.